src/Tools/a2md/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.CodeAnalysis.Analyzers" version="1.0.0" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.Common" version="1.0.0" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.CSharp" version="1.0.0" targetFramework="net46" />
+  <package id="Microsoft.CodeAnalysis.VisualBasic" version="1.0.0" targetFramework="net46" />
+  <package id="System.Collections.Immutable" version="1.1.36" targetFramework="net46" />
```

Comment:
do you actually consume this package or did you copy a packages.config from elsewhere?